screwdriver.yaml

```diff
@@ -2,20 +2,28 @@ workflow:
     - publish
 
 shared:
-    image: node:6
+    image: centos
 
 jobs:
     main:
         steps:
-            - install: npm install
-            - test: npm test
-
+            # To verify if the hyperctl-download script works
+            - hyperctl-download: bash -xe scripts/hyperctl-download.sh
+            - chmod: chmod +x ./hyperctl
+            - test-hyperctl: ./hyperctl info -h
     publish:
         steps:
-            - install: npm install semantic-release
-            - publish: npm run semantic-release
+            - hyperctl-download: bash -xe scripts/hyperctl-download.sh
+            - chmod: chmod +x ./hyperctl
+            - test-hyperctl: ./hyperctl info -h
+            - setup-ci: git clone https://gist.github.com/3d2388b2a7ba658cdcdaffa8cd874e50.git ci
+            - tag: ./ci/git-tag.sh
+            - release: ./ci/git-release.sh
+        environment:
+            DOCKER_REPO: screwdrivercd/hyperctl-image
```

Comment:
I don't see this environment value being used in either the `tag` or `release` steps. Is it being used by the `hyperctl-download.sh`?